src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <mapped-superclass name="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroup"
+                       repository-class="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupRepository"
+                       table="at_target_groups">
+
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="title" column="title" type="string" length="255" nullable="false"/>
+        <field name="description" column="description" type="text" nullable="true"/>
+        <field name="priority" column="priority" type="integer" nullable="false"/>
+        <field name="allWebspaces" column="allWebspaces" type="boolean" nullable="false"/>
+        <field name="active" column="active" type="boolean" nullable="false"/>
+
+        <one-to-many field="webspaces"
+                     mapped-by="targetGroup"
+                     target-entity="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupWebspace">
+            <cascade>
+                <cascade-all/>
```

Comment:
Sorry, my bad 🙈 You can keep the cascade persist, but instead of cascade-remove you should use the `onDelete` option to enforce this on a database level as described in the [doctrine documentatino](http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/working-with-objects.html#removing-entities)